var/spack/repos/builtin/packages/py-gssapi/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGssapi(PythonPackage):
+    """Python-GSSAPI provides both low-level and high level wrappers
+    around the GSSAPI C libraries."""
+
+    homepage = "https://github.com/pythongssapi/python-gssapi"
+    pypi = "gssapi/gssapi-1.8.2.tar.gz"
+
+    maintainers = ["wdconinc"]
```

Comment:
```suggestion
    maintainers("wdconinc")
```
This new formats appends instead of overwrites so I will still be listed